proca/config/releases.exs

```diff
@@ -118,6 +118,9 @@ config :proca, Proca,
 
 config :proca, Proca.Supporter, fpr_seed: System.get_env("FINGERPRINT_SEED", "")
 
+config :proca, Proca.Server.MTTWorker,
+  max_messages_per_cycle: String.to_integer(System.get_env("MAX_MESSAGES_PER_CYCLE", "99"))
```

Comment:
What is the duration of a cycle?

Reply:
cycle repeats after 30 seconds after last worker finished, so under heavy load cycles can be longer